### PR TITLE
Fix PHP 8.1 shell helper fatal error

### LINKED ISSUE #798: Generation of shell helper: Fatal error
> bin/cake bake shell_helper BazBaz

in PHP 8.1:

```
PHP Fatal error:  Declaration of App\Shell\Helper\BazBazHelper::output($args) must be compatible with Cake\Console\Helper::output(array $args): void in /var/work/devilbox/data/www/sandbox/src/Shell/Helper/BazBazHelper.php on line 20

Fatal error: Declaration of App\Shell\Helper\BazBazHelper::output($args) must be compatible with Cake\Console\Helper::output(array $args): void in /var/work/devilbox/data/www/sandbox/src/Shell/Helper/BazBazHelper.php on line 20
```

### PR DESCRIPTION
**Pull request by LordSimal**
fixes #798

### CODE REVIEW

File: templates/bake/Shell/helper.twig [hunk 1]
```diff
@@ -32,7 +32,7 @@ class {{ name }}Helper extends Helper
      *
      * @return void
      */
-    public function output($args)
+    public function output(array $args)
```
Comment (dereuromark) [inline templates/bake/Shell/helper.twig]: Should we add void return too?

Comment (LordSimal) [reply templates/bake/Shell/helper.twig]: ah right, would be nice as well 😄

**Comment by dereuromark** [comment]
This should probably have been fixed in 2.x instead.

**Comment by LordSimal** [comment]
oh, sorry didn't know that

**Comment by dereuromark** [comment]
2.next is for New features or bc breaking issues

**Pull request merged**